Applications/Spire/Source/KeyBindings/RegionTerm.cpp

```diff
@@ -0,0 +1,22 @@
+#include "Spire/KeyBindings/RegionTerm.hpp"
+#include <algorithm>
+
+using namespace Nexus;
+using namespace Spire;
+
+RegionTerm::RegionTerm(Region region)
+  : m_region(std::move(region)) {}
+
+bool RegionTerm::is_satisfied(const std::vector<KeyBindings::Tag>&
+    tags) const {
+  auto it = std::find_if(tags.begin(), tags.end(), [] (auto& tag) {
+    return tag.get_key() == 55;
+  });
+  if(it != tags.end()) {
+    auto value = it->get_value<Region>();
+    if(value) {
+      return *value <= m_region;
+    }
+  }
+  return m_region.IsGlobal();
```

Comment:
If no region is present this term shall not be satisfied.

Reply:
Does it also apply if a region tag is present but its value is not set? Or should it be interpreted as a global region?

Reply:
We might change this later depending on feedback from users, but for now I suspect if a region is present but unset, it shall not satisfy this term.